NosTests/Controller/NoteEditorControllerTests.swift

```diff
@@ -0,0 +1,218 @@
+import XCTest
+import UIKit
+
+// swiftlint:disable implicitly_unwrapped_optional
```

Comment:
If I remember correctly from a previous project, we could configure SwiftLint differently for files in NosTests. I don't know if I want to encourage that -- I consider test code to be the same as production code -- but it would certainly make things like this a little easier. Just something to think about. I'm not advocating for doing it now, and definitely not in this PR.

Reply:
Oh I never thought about that but it makes sense. I want our test code to read nicely and often `let force = unwrap!` is easier to read than an `if let` or `guard` and I don't mind if the test target crashes when tests fail. I know some people don't feel the same way. Something to revisit maybe.

Reply:
I do love `XCTUnwrap`. I wonder if we could use that in a computed property... 🤔

Anyway, as I said before, I'm good with this as-is for now.